Makefile

```diff
@@ -716,20 +722,16 @@ generate_yaml_coredump:
 #             as a followup PR.
 generate_e2e_test_yaml: $(HELM) $(HOME)/.helm helm-repo-add istio-init.yaml
 	$(HELM) dep update --skip-refresh install/kubernetes/helm/istio
-	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+	#./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
```

Comment:
updateVersion.sh I believe uses values-* to generate manifests for test cases.

Reply:
The copy paste below also generates the same stuff, but I will remove it and try again.